api/service_instance_controller.go

```diff
@@ -83,3 +101,48 @@ func (c *ServiceInstanceController) Routes() []web.Route {
 		},
 	}
 }
+
+func (c *ServiceInstanceController) GetParameters(r *web.Request) (*web.Response, error) {
```

Comment:
perhaps we should block async=true API calls to this method as it works only in the sync mode & return async is not supported error.